hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -226,6 +227,9 @@ private void createMemState(TimedMemberState timedMemberState, MemberStateImpl m
                     count = handleQueue(memberState, count, config, ((QueueService) service).getStats(), longInstanceNames);
                 } else if (service instanceof TopicService) {
                     count = handleTopic(memberState, count, config, ((TopicService) service).getStats(), longInstanceNames);
+                } else if (service instanceof ReliableTopicService) {
```

Comment:
We should eventually introduce an interface 'StatisticsProvider' and let every service that provides statistics implement this.